### PR TITLE
[W6.4h][W13-B2]Bay Wei Heng

### PR DESCRIPTION
**Pull request by bayweiheng**
LO-Interfaces

### CODE REVIEW

File: src/seedu/addressbook/data/person/Printable.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Printable.java:3]: This interface and its methods should have header comments. 
If not, developers who need to implement this in their class will not know what the expected behavior is.

File: src/seedu/addressbook/ui/Formatter.java [hunk 2]
```diff
@@ -63,4 +64,14 @@ private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/Formatter.java:68]: What would be a better place to have this method?

**Comment by okkhoy** [comment]
@bayweiheng 
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/data/person/Printable.java (hunk 1 above)
Comment (bayweiheng) [reply src/seedu/addressbook/data/person/Printable.java:3]: Ok, will definitely take note of this when doing my Phase B project! Thank you!

File: src/seedu/addressbook/ui/Formatter.java (hunk 2 above)
Comment (bayweiheng) [reply src/seedu/addressbook/ui/Formatter.java:68]: I'm not sure how to decide, to be honest. Was choosing between ReadOnlyPerson and Formatter...

**Pull request closed without merging**